### ISSUE TITLE
Aggregate data by ward

### ISSUE THREAD

**Issue by thcrock** [open-city/recycling#130]
It would be useful to see what aldermen could have the biggest impact on enforcing this law. By linking these addresses against ward shapefiles, this can be figured out and displayed in some way, and provide a useful place for aldermen to look and see a snapshot of their ward.

**Comment by alexkahn** [comment]
We currently have something like this but are currently updating the ward data. See this page [here](http://mybuildingdoesntrecycle.com/wards).

**Comment by alexkahn** [comment]
We'll probably want to make this generally accessible but since the data is stale that will be another issue.

**Comment by thcrock** [comment]
That's true, some of the data could be old enough so as to be no longer true. You could show the newest data by ward, but that's a bit misleading since even the oldest reports could still be true.

**Comment by alexkahn** [comment]
Just referring to the ward boundaries and contact data, we're treating all reports as equal and valid.

**Comment by alexkahn** [comment]
@matiaskitty Do we want to put in a nav button for this view or should there be some other way of viewing these data?
